server/pbenchinacan/run-pbench-in-a-can

```diff
@@ -1,5 +1,25 @@
 #!/bin/bash
 
+#
+# Specification for a self-contained Pbench Server Environment.
+#
+# This configuration creates a pod for the Pbench Server deployment using
+# four separate containers: the Pbench Server and its associated
+# Elasticsearch instance, PostgreSQL database instance, and KeyCloak broker
+# instance. It then runs the server/pbenchinacan/load_keycloak.sh to
+# configure the keycloak broker for the minimum requirements of the
+# Pbench-Server.
+#
+# The pod should be invoked via the following command
+#    server/pbenchinacan/run-pbench-in-a-can
+#
+# The resulting pod maps the following ports:
+#
+#   5432 - PostgreSQL
+#   8080 - HTTP web server (and Pbench Dashboard)
+#   8090 - keycloak authentication server
+#   9200 - Elasticsearch
+
 export PB_CONTAINER_REG=${PB_CONTAINER_REG:-"images.paas.redhat.com"}
 export PB_ORG_NAME=${PB_ORG_NAME:-"pbench"}
```

Comment:
I strongly suggest pulling all four images from the same registry and organization.  And, I wouldn't bother parameterizing the organization.

So, I suggest replacing the definition of `PB_ORG_NAME` with
```
export PB_CONTAINER_IMAGE_PREFIX=${PB_CONTAINER_REG}/pbench
```
and then using that to reference each of the images.

In particular, the Elasticsearch image reference seems to be missing its registry, and the Keycloak image is hard-coded to `Quay.io` which I think is not what we want.